json-lint/schemas/inventory/main.json

```diff
@@ -27,8 +27,19 @@
             "$id": "/properties/placements",
             "type": "object",
             "title": "Placements Object",
-            "description": "contain the placements with keys like : \n - nativeAd \n - adsBelowTopMedia \n - topMosaicAd \n - inline \n - inline_5",
-            "additionalProperties": true
+            "description": "contain the placements with keys like: \n - sticky \n - inter \n - top \n - top_mosaic \n - top_details \n - bottom_mosaic \n - bottom_details_2 \n - inline \n - inline_5 \n ...",
+            "patternProperties": {
+                "(top|inline|bottom)_*(mosaic|details)*_*([a-zA-Z0-9]+)*_*([0-9]+)*": {
```

Comment:
this match everything...
topIMSUPPERCRAZY ⚠️ 

Add tests of wrong also, to make sure we don't accept wrong cases

Reply:
@joaquimsantmrf 
I've added wrong cases already but this will match things like this:
`bottom_inline_3`
`bottom_whatever_whatever_3`
`inline_inline_mosaic_39483`
...

In other words, as long as the property string starts with either top/inline/bottom, the check will proceed on for object validation in the placements/schema.json.

The elasticity of this is because we could have cases like:
`top_mosaic_home_1`
`top_mosaic_1`
`top_home_1`
`top_mosaic`
`top_home`
`top_1`

As I specified in the description of this PR, I can restrict the patterns to match each combination singularly. If this is what we want.

Reply:
As discussed offline, since it would not be possible to cover all the cases, this regex would not be different from `(top|inline|bottom)(.*)*`, so I would use this which is shorter and easier to read. The purpose of the regex is to validate, not to document.

Reply:
@lucacamplimrf That's right.

Even more, even if I could put down the list of all the combos, there will always be at the very end a regexp like the one we specified, to cover cases of optional key parts (navigationLevel, sectionName, virtualPage number). But then all the other specified cases would just be redundant.